SlowCheetah.VisualStudio/Utilities/NugetHandler/NugetPackageHandlerBase.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using NuGet.VisualStudio;
+
+namespace SlowCheetah.VisualStudio
+{
+    public abstract class NugetPackageHandlerBase : INugetPackageHandler
+    {
+        protected IServiceProvider package;
+        protected const string pkgName = "SlowCheetah";
```

Comment:
this should be a resource or retrieved from somewhere else. Also `const` should start with upper-case.